.github/workflows/lint.yml

```
@@ -10,7 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
```

We should use `v3` to spare dependabot some work.
```suggestion
      - uses: actions/setup-go@v3
```